### DISCUSSION TITLE
Developer Portal - ENUM displayed in Table View

### DISCUSSION THREAD

**Discussion by BenCloud75** [Azure/api-management-developer-portal#1474] [General]
Hello, 

I would like to escalade a missing feature of the AZURE Portal. Indeed, it seems the possible list of "ENUM" are not listed in the view "Table View" althought they are mentionned in the "Raw Schema View". 

I attach two pictures, one of the RawSchemaView displaying 2 enums "Private" and "Professionnal", and the same entry in a TableView format. This cause some confusion with our API Consumers as it is not obvious for them to click on "RawSchemaView" to access per default to the full description. 

**Raw Schema View (with ENUM)**
<img width="257" alt="Raw Schema View" src="https://user-images.githubusercontent.com/75790837/135825482-6eab0167-e69d-4fd3-ac75-0833f7f1ba73.png">

**Table View (without ENUM)**
<img width="549" alt="TableView" src="https://user-images.githubusercontent.com/75790837/135825485-e2af8c21-4eda-4fb6-98dc-3196c4568fee.png">
.

**Comment by mikebudzynski** [comment]
@bvallet75, please post your comment in an existing Issue or open a new Issue with the bug report.

**Reply by BenCloud75** [reply]
Hello @mikebudzynski , done here [[Issue #1478](https://github.com/Azure/api-management-developer-portal/issues/1478)]
